cmd/juju/bootstrap_test.go

```diff
@@ -321,6 +322,10 @@ func (s *BootstrapSuite) checkSeriesArg(c *gc.C, argVariant string) *cmd.Context
 // In the case where we cannot examine an environment, we want the
 // error to propagate back up to the user.
 func (s *BootstrapSuite) TestBootstrapPropagatesEnvErrors(c *gc.C) {
```

Comment:
this is very platform specific. When/if we will support windows state machine, this will probably be rewritten.